selectors/local_storage.ts

```diff
@@ -57,5 +57,5 @@ export const getPreviousTeamLastViewedType = (state: GlobalState) => {
     const previousTeamID = getPreviousTeamId(state);
     const userId = getCurrentUserId(state);
 
-    return localStorageStore.getPreviousViewedType(userId, previousTeamID, state);
```

Comment:
The localStorageStore is also converted to typescript. It wraps localStorage apis to get and set item, methods which only accepts strings. I thought it was acceptable if the team didn't exist to pass empty string instead of `undefined`.